README.md

```diff
@@ -1,73 +1,68 @@
 # Nova quickstart
 
-This is a simple example of a nov base Discord bot inspired by the discord.js structure.
+This document illustrates the process of creating a Discord Bot using the 
+Nova platform.
+The Bot interacts with Nova using a library following the popular 
+[Discord.js](https://discord.js.org) structure.
 
-## What you'll be doing
-
-This repo is a simple example of a bot developped using Nova. It's a simple bot that executes `ping` (ICMP Echo) requests using slash-commands.
-
-> Issuing arbitrary pings is not viable for a real discord bot, this is an example. Acces to your developement bot should be kept private to avoid abuse.
+> Credentials should always be private.
 
 <img src="imgs/nova1.png">
+![An example of a Ping command, calculating the latency in a loca network.](imgs/nova1.png)
 
-## Running
+## Getting Started
 
-### Configuring your instance
+### Configuring the Bot
 
-> The nova-all-in-one we are going to use is going to search this directory for a `default.{toml,json,json5,yml,yaml}`, feel free to use the configuration language you feel confortable with.
+Configuration is done by creating a file inside the [config](config/) directory.
+The file should be named "default", we support TOML, JSON (and JSON5) and YAML.
 
-You need to fill the configuration file and rename it to `default.yml` in the `config/` folder.
+Make sure to configure properly your instance before getting started, see the
+example file in the directory for more information.
 
-### Getting nova running
+### Starting the Bot
 
-Download the nova-all-in-one server and put it in `bin/nova`.
-You can find the binary in the nova project releases page.
+You need to install Nova.
+For simplicity we will use the [`all-on-one`](https://github.com/discordnova/Nova/tree/main/exes/all)
+distribution, which provides a single binary. 
+This is usesul in development environments.
+Place the binary inside the [`bin`](bin/) directory, under the name `nova`.
 
-> You can also use a full Nova instance running in `docker-compose` or an orchestrator such as [Kubernetes](https://kubernetes.io). Using a real nova instance, you'll need to store the configuration file safely depending on your orchestrator.
+> To get the most out of Nova, we strongly encourages you to deploy Nova 
+correctly (as microservices), on the orchestrator you want (Kubernetes, 
+Docker Compose...).
+> The all-in-one distribution IS NOT MADE for production environments.
```

Comment:
and is adequate for development environments

Reply:
This would be a repetition from the point said earlier

> For simplicity we will use the [`all-on-one`](https://github.com/discordnova/Nova/tree/main/exes/all)
distribution, which provides a single binary. 
This is usesul in development environments.